PixiEditor/Views/MainWindow.xaml

```diff
@@ -83,6 +83,8 @@
                               Command="{Binding FileSubViewModel.SaveDocumentCommand}" CommandParameter="AsNew" />
                     <MenuItem Header="_Export" InputGestureText="Ctrl+Shift+Alt+S" Command="{Binding FileSubViewModel.ExportFileCommand}" />
                     <Separator />
+                    <MenuItem Header="_Settings" Command="{Binding MiscSubViewModel.OpenSettingsWindowCommand}" />

```

Comment:
I would expect it to be in the `Edit` tab

Reply:
I was thinking about that, some software has this in `File` some in `Edit`. Not sure which one should we use.

Reply:
Ok, I moved it to `Edit`